afl/src/bin/cargo-afl.rs

```diff
@@ -224,11 +244,27 @@ where
     I: IntoIterator<Item = S>,
     S: AsRef<OsStr>,
 {
-    let cmd_path = common::afl_dir(None).join("bin").join(tool);
-    let mut cmd = Command::new(cmd_path);
-    cmd.args(args);
+    let cmd_path: PathBuf;
+    let mut cmd;
+    if tool.to_string() == "sudo".to_string() {
```

Comment:
Could we structure the code involving this `if` block as follows?
```rust
let mut cmd = if tool == "afl-system-config" {
    // sudo stuff
} else {
   ...
};
cmd.args(args);
// Rest as before.
```
Also, could we please add `--reset-timestamp` to the sudo invocation? This way, if the user is surprised by the sudo invocation, they have a chance to cancel it.